hoomd/hpmc/ShapePolyhedron.h

```diff
@@ -710,7 +710,7 @@ DEVICE inline bool test_narrow_phase_overlap( vec3<OverlapReal> dr,
         for (unsigned int j= 0; j< nb; j++)
             {
             unsigned int nverts_b, offs_b;
-            unsigned int jface = b.tree.getParticle(cur_node_b, j);
+            unsigned int jface = b.tree.getParticleByNode(cur_node_b, j);
```

Comment:
I can't seem to find the addition or change of `getParticleByNode` was `getParticle` just an alias to `getParticleByNode` and we are being more explicit now, or did I just miss this in an earlier commit?

Reply:
there are now two ways to access the particle list, `getParticleByNode` and `getParticleByIndex` (for faster iteration), used in `jit/EvaluatorUnionGPU.cuh`

Reply:
Ah, thanks for the clarification, @jglaser.

Reply:
This changed was introduced in #549. Due to a situation where ShapePolyhedron.h somehow got converted to 5-space indentation on `feature/new-object-API` and back to 4-space indentation in this PR, the conflicts cause this to appear as a change instead of a merge.